airflow/models/dagrun.py

```diff
@@ -87,6 +87,16 @@ def state(self):
         return synonym('_state',
                        descriptor=property(self.get_state, self.set_state))
 
+    def get_conf(self):
```

Comment:
```suggestion
    def _get_conf(self):
```

to make this "private"